redash/models.py

```diff
@@ -658,10 +660,9 @@ def all_queries(cls, groups, drafts=False):
             .order_by(cls.created_at.desc())
 
         if drafts:
-            q = q.where(Query.name == 'New Query')
+            q = q.where(Query.is_draft == True)
         else:
-            q = q.where(Query.name != 'New Query')
-
+            q = q.where(Query.is_draft == False)
```

Comment:
I agree it'd be better to ensure that newly created queries are marked as drafts. I've updated this below.